src/main/java/sirius/web/data/ExcelExport.java

```diff
@@ -399,6 +402,20 @@ public ExcelExport addRow(Object... row) {
      * @return the export itself for fluent method calls
      */
     public ExcelExport addRowAsList(Collection<?> row) {
+        if (rows > workbook.getSpreadsheetVersion().getMaxRows() - 1) {
+            return this;
+        }
+        if (rows == workbook.getSpreadsheetVersion().getMaxRows() - 1) {
+            if (Strings.isFilled(getMaxRowsReachedMessage())) {
+                Row r = currentSheet.createRow(rows);
+                addCell(r, getMaxRowsReachedMessage(), 0, normalStyle);
+            }
+            rows++;
+            if (maxRowsReachedHandler != null) {
+                maxRowsReachedHandler.accept(currentSheet.getSheetName());
+            }
+            return this;
+        }
```

Comment:
shoulden the whole block be in the if below or shouldn't this return immediatelly?

Reply:
I dont think so. Why do you think so?

Reply:
has been moved